package.json

```diff
@@ -78,14 +78,16 @@
     "@ngtools/webpack": "^11.0.0",
     "@playwright/test": "^1.18.1",
     "@types/chart.js": "^2.9.24",
-    "@types/d3": "^5.7.2",
+    "@types/d3": "^7.4.0",
+    "@types/d3-selection": "^3.0.4",
```

Comment:
This doesn't seem like it's needed since @types/d3 exports the types from the other @types/d3-* packages.